packages/client/modules/meeting/components/TemplateScaleValueList.tsx

```diff
@@ -7,23 +7,25 @@ import {TemplateScaleValueList_scale} from '~/__generated__/TemplateScaleValueLi
 import useAtmosphere from '../../../hooks/useAtmosphere'
 import useMutationProps from '../../../hooks/useMutationProps'
 import MovePokerTemplateScaleValueMutation from '../../../mutations/MovePokerTemplateScaleValueMutation'
+import AddScaleValueButtonInput from './AddScaleValueButtonInput'
 import TemplateScaleValueItem from './TemplateScaleValueItem'
 
 interface Props {
   scale: TemplateScaleValueList_scale
 }
 
 const ScaleList = styled('div')({
-  margin: 0,
```

Comment:
divs will never have margin or padding by default, so unless you're inheriting another styled component you won't need these